.github/workflows/actions.yml

```
@@ -1,7 +1,7 @@
 name: CI
 
 on:
-  push:
+  workflow_dispatch:
```

```suggestion
  push:
    - main
    - feat/dex_v2
```
The `push` event is required on the base branch of the PR, otherwise the report is not generated.